src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return price * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
You can remove the condition `days < 7` from the second `if` statement. Since the code returns early if `days` is less than 3, and the last condition checks for `days >= 7`, this check is redundant.